src/profile/supporting_types/account_for_display.rs

```diff
@@ -13,10 +13,12 @@ use crate::prelude::*;
     derive_more::Display,
     uniffi::Record,
 )]
-#[display("{label} | {address}")]
+#[display("{display_name} | {address}")]
 pub struct AccountForDisplay {
     pub address: AccountAddress,
-    pub label: DisplayName,
+
+    #[serde(rename = "label")]
+    pub display_name: DisplayName,
```

Comment:
Never liked this inconsistency, the JSON we will live with (if we dont change CAP21), but at least the field can have the same names as the rest..